### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by chukwuebukaVictor**
HELLO_MICROVERSE

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
```
Comment (bellom) [inline .github/workflows/linters.yml:32]: If you look at the end of this line you will see `.hintrc`

Let's go back to the [Linter Setup](https://github.com/microverseinc/linters-config/tree/master/html-css) and add `.hintrc` file to your repo

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
```
Comment (bellom) [inline .github/workflows/linters.yml:46]: If you look at the end of this line you will see `.stylelintrc.json `

Let's go back to the [Linter Setup](https://github.com/microverseinc/linters-config/tree/master/html-css) and add `.stylelintrc.json ` file to your repo

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @chukwuebukaVictor  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

1. Kindly check the comments under the review.
2. To make your commit messages look more professional, you should always capitalize them.
3. Good skills showing you know how to use `.gitignore` but in this project, you're supposed to add the text `"This file should be ignored by git"` inside the `test.md` file.
4. Nice work having at least a word PR description, You could be more specific in your PR description by stating exactly what you did.

 PR summary should be well detailed.
 
Example; 
image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chukwuebukaVictor** [comment]
Effected the changes

File: README.md [hunk 3]
```diff
@@ -1 +1,81 @@
...
+Additional description about the project and its features.
+
+## Built With
+
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```
Comment (carlos-ssh) [inline README.md:18]: [OPTIONAL]
- Your live demo isn't working

File: README.md [hunk 4]
```diff
@@ -1 +1,81 @@
...
+    VS Code
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
```
Comment (carlos-ssh) [inline README.md:48]: - Write a descriptive and easy to understand README in business English for every project submitted.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @chukwuebukaVictor , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
Kindly check the comments below 👇🏼

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chukwuebukaVictor** [comment]
I have effected my corrections, please kindly review.

**Review by carlos-ssh — APPROVED** [review]
Hi @chukwuebukaVictor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**